.gitignore

```diff
@@ -39,5 +39,6 @@ testem.log
 Thumbs.db
 
 .nx/cache
+.nx/workspace-data
```

Comment:
We can replace both of these `.nx` ignored folders with `.nx/**`

Reply:
Also, let's please include the target folder in the git ignore

[Reference](https://git-scm.com/docs/gitignore)